src/Compilers/CSharp/Test/Emit/Attributes/AttributeTests_Nullable.cs

```diff
@@ -3228,7 +3228,7 @@ class Program
             static void validate(ModuleSymbol module)
             {
                 var globalNamespace = module.GlobalNamespace;
-                VerifyBytes(globalNamespace.GetMember<FieldSymbol>("Program.F0").TypeWithAnnotations, new byte[] { 0 }, new byte[] { }, "System.ValueTuple");
+                VerifyBytes(globalNamespace.GetMember<FieldSymbol>("Program.F0").TypeWithAnnotations, new byte[] { 0 }, new byte[] { 0 }, "System.ValueTuple");
```

Comment:
what is denoted by the newly introduced `0` in the metadata?

Reply:
There is an encoding position for the element name.

Reply:
I don't get it.  Where could there be an element name?

---
In reply to: [359499366](https://github.com/dotnet/roslyn/pull/40459#discussion_r359499366) [](ancestors = 359499366)

Reply:
My bad. Fixed